UPGRADE-3.1.md

```diff
@@ -107,7 +107,7 @@ FrameworkBundle
    framework:
        cache:
            pools:
-               serializer:
+               cache.serializer:
```

Comment:
because I've been told that Symfony core prefers explicit over implicit & "magic" conventions. This in fact is much more flexible and doesn't force any service prefix. See examples above where an `app.doctrine_cache` pool is created. We don't have to teach any special convention here. Just give a service name.